library/xilinx/axi_adxcvr/bd/bd.tcl

```diff
@@ -0,0 +1,17 @@
+
+proc init {cellpath otherInfo} {
+  set ip [get_bd_cells $cellpath]
+
+  bd::mark_propagate_override $ip " \
```

Comment:
Should somebody be able to manually overwrite this? If they should not be overwritten use `mark_propagate_only`.